packages/oui-file/_variables.less

```diff
@@ -0,0 +1,37 @@
+@import "../oui-color/_constants";
+
+@oui-file-margin: 0 0 rem-calc(20);
+
+@oui-file-placeholder-font-color: #b3b3b3;
+
+@oui-file-icon-width: rem-calc(40);
+@oui-file-icon-color: @oui-color-sapphire;
+@oui-file-icon-color_error: @oui-color-error-dark;
+@oui-file-icon-font-size: rem-calc(12);
+
+@oui-file-droparea-height: rem-calc(80);
+@oui-file-droparea-font-color: @oui-color-sapphire;
+@oui-file-droparea-font-color_hover: @oui-color-silver;
+@oui-file-droparea-border: dashed 1px @oui-color-pacific;
+@oui-file-droparea-border_hover: dashed 1px @oui-color-silver;
+@oui-file-droparea-background-color: @oui-color-polar;
+@oui-file-droparea-background-color_hover: @oui-color-alabaster;
+
+@oui-file-heading-margin: rem-calc(15) 0 rem-calc(5);
+@oui-file-heading-font-color: @oui-color-sapphire;
+@oui-file-heading-font-size: rem-calc(14);
+
+@oui-file-preview-size: rem-calc(78);
+@oui-file-preview-icon-size: rem-calc(20);
+
+@oui-file-attachment-height: rem-calc(40);
+@oui-file-attachment-line-height: rem-calc(20);
+@oui-file-attachment-padding: rem-calc(10) rem-calc(40) rem-calc(10) rem-calc(20);
+@oui-file-attachment-font-color: @oui-color-sapphire;
+@oui-file-attachment-font-color_disabled: @oui-color-silver;
+@oui-file-attachment-font-color_error: @oui-color-error-dark;
+@oui-file-attachment-font-size: rem-calc(14);
+@oui-file-attachment-border: solid 1px @oui-color-silver;
+@oui-file-attachment-border_error: solid 1px @oui-color-error-medium;
+@oui-file-attachment-background: #fff;
```

Comment:
No `oui-color-white` ? :D

Reply:
It's mostly because I think `white` is easier to understand than `#fff`

Reply:
If you're doing LESS, you normally understand what `#fff` is.
Maybe I missed the `/s` in my upper comment :)